CRADLE/correctbiasutils/__init__.py

```diff
@@ -27,7 +27,15 @@
 SCATTERPLOT_SAMPLE_COUNT = 10_000
 SONICATION_SHEAR_BIAS_OFFSET = 2
 
-CORRECTED_RC_TEMP_FILE_STRUCT_FORMAT = "@5sLLf"
+CORRECTED_RC_TEMP_FILE_STRUCT_FORMAT = "=LLf"
```

Comment:
what does this mean?

Reply:
These are format strings used by the python [struct](https://docs.python.org/3/library/struct.html) library for packing data into `bytes` objects/C structs.

`@5sLLf` means 5 characters, two long ints, and one float with native byte-order (endian-ness) and size.

`=LLf` means two long ints and one float with native byte-order and "standard" (32-bit) size.

Reply:
5sLLF represented chromosome name/start/end/value data
LLF represents start/end/value data

I dropped the chromosome characters because I no longer needed to include them, since they're in the file name.